api/v1alpha1/httproute_types.go

```diff
@@ -190,6 +200,75 @@ type HTTPRouteAction struct {
 	Extension *core.TypedLocalObjectReference `json:"extension"`
 }
 
+// TLSConfig defines the schema of a TLS configuration.
+type TLSConfig struct {
+	// Termination defines how to terminate TLS connections.
+	//
+	// If unspecified, TLS termination type "Edge" will be used.
+	//
+	// Support: core
+	//
+	// +optional
+	Termination TLSTerminationType `json:"termination,omitempty"`
+
+	// Certificate is a reference to a Kubernetes object containing
+	// the identity certificate, key and CA certificate used to terminate
+	// the TLS connection. Certificate is used with "Reencrypt" and "Edge"
+	// TLS termination types.
+	//
+	// If apiGroup and kind are empty, will default to a Kubernetes Secret
+	// resources.
+	//
+	// If unspecified, the certificate of the binding gateway will be used.
+	//
+	// Support: Core (Kubernetes Secrets)
+	// Support: Implementation-specific (For other resource types)
+	//
+	// +optional
+	Certificate core.TypedLocalObjectReference `json:"certificate,omitempty"`
+
+	// DestinationCACertificate is a reference to a CA certificate used
+	// for establishing a TLS connection with the final destination when
+	// using TLS termination type "Reencrypt".
```

Comment:
This is a CA bundle? Then we can call this `CACertificates` and I'd suggest the comment read something more like:
```
CACertificates is a reference to the bundle of trusted CA certificates used to validate the server certificate when establishing a TLS connection ...
```